cargo-flash/src/main.rs

```diff
@@ -41,6 +41,8 @@ struct Opt {
         long = "chip-description-path"
     )]
     chip_description_path: Option<String>,
+    #[structopt(name = "nrf-recover", long = "nrf-recover")]
+    nrf_recover: bool,
```

Comment:
I think we should do this in more generic fashion. For now this is okay but for later on I would prefer a generic `--unlock`.